src/components/Dropdown.tsx

```diff
@@ -8,6 +8,8 @@ import { useKeyPress } from '../hooks/useKeyPress';
 import { useKeyboard, KeyCode } from '../hooks/useKeyboard';
 
 import { Popup } from './Popup';
+import Input from './Input';
```

Comment:
We must delete default exports at all from package

Reply:
I deleted default export of Dropdown, but I believe it should be another issue - #223